### ISSUE TITLE
crond fails to run because root's password is expired

### ISSUE THREAD

**Issue by alexskr** [ncbo/virtual_appliance#16]
The last step in the appliance build process is to clean up and seal the appliance where we expire root's password in order to force root password change on the first login.  This breaks 
crond.

cron log:
```
(root) PAM ERROR (Authentication token is no longer valid; new one required)
(root) FAILED to authorize user with PAM (Authentication token is no longer valid; new one required)
```

**Comment by alexskr** [comment]
3 possible solutions:
1. switch from root to centos user for SSH logins similar to how we do for AMIs
2. change pam.d https://access.redhat.com/solutions/36657
3. run mgrep dictionary refresh cronjob as ontoportal user.

**Comment by alexskr** [comment]
implemented #1 and #3 option in appliance v3.0.1
